src/template.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+    />
+    <script
+      defer
+      src="https://kit.fontawesome.com/667d7efdcc.js"
+      crossorigin="anonymous"
+    ></script>
+
+    <title>Movie Slider</title>
+  </head>
+  <body>
+    <div class="container">
+      <div class="navbar">
+        <h2 class="logo">Movie concert</h2>
+        <ul class="nav-list">
+          <li class="nav-item movies">Movies</li>
+          <li class="nav-item">TV</li>
+          <li class="nav-item">About</li>
+        </ul>
+      </div>
+      <div class="cards-container"></div>
+      <div class="modal">
+        <div class="modal-content">
+          <div class="image">
+            <img src="" alt="" class="film-image">
+            <h2 class="name"></h2>
+            <div class="details"></div>
+          </div>
+         
+          <p class="close">&#x2715</p>
+          <div class="comment-card">
+            <form>
+              <label for="username">Username:</label>
+              <input type="text" id="username" name="username" />
+
+              <label for="comment">Comment:</label>
+              <textarea id="comment" name="comment"></textarea>
+
+              <button id="submit" class="submit">Submit</button>
+            </form>
+          </div>
+          <div class="comment-num">
+            <h3 class="number">Comment</h3>
+          </div>
+          <div class="comment-section">
+           <h4>Comment</h4>
+          </div>
+        </div>
+      </div>
+    </div>
+    <footer>&#169; Created by Rabaya and Hussein under the CC license</footer>
+  </body>
```

Comment:
- [ ]  Your home page footer should be similar to the footer from the mockup.  Your footer content is aligned in the center while it should be align in the left side as it is provided in the mockup. 
Your footer content 
![image](https://github.com/husush23/capstone--2/assets/43877689/ccda698a-0470-438e-90bb-bd3da28cec8d)

provided footer 
![image](https://github.com/husush23/capstone--2/assets/43877689/aa976393-6249-4d6a-8873-eb0dab49a071)